.github/workflows/python.yml

```diff
@@ -0,0 +1,21 @@
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - "*"
+jobs:
+  test:
+    name: Ex3 Linux
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: goanpeca/setup-miniconda@v1
+        with:
+          activate-environment: louis
+          environment-file: environment.yml
+          auto-activate-base: false
+      - shell: bash -l {0}
+        run: |
+          cd $GITHUB_WORKSPACE && python -m compileall .
```

Comment:
We will probably need to run tests here in the future rather than only compiling. But, it looks good to me.